.env.example

```diff
@@ -1,7 +1,7 @@
 BCROS_BASE_URL = "http://localhost:8080/"
 AUTH_URL = "https://dev.bcregistry.ca/business/auth/"
 NAME_REQUEST_URL = "https://dev.bcregistry.ca/namerequest/"
-BUSINESS_URL = "https://dev.bcregistry.ca/business/"
+BUSINESS_URL = "https://dev.bcregistry.ca/business/auth/"
```

Comment:
The old URL was actually Filings UI, which, after loading and not seeing a business ID, redirected to the auth pages. The new URL is more direct.

Reply:
@doug-lovett Can you please update ^^^ in the deployed environments?

Reply:
> @doug-lovett Can you please update ^^^ in the deployed environments?

@doug-lovett Thanks for updating Dev! I still see an old URL in https://www.bcregistry.daxiom.ca/. Can this be updated, too?